source/configuration_manual/mail_location/obox/compression.rst

```diff
@@ -41,10 +41,10 @@ compressed to fscache. On one hand this increases the fscache's file, but on
 the other hand it requres Dovecot to always uncompress the files before
 accessing them.
 
-This uncompression means that the file is temporarily written to
-``mail_temp_dir``. By using the ``compress`` fs wrapper in ``obox_fs`` line the
-mails are stored uncompressed in fscache, and reading the mails from there
-doesn't require writing to ``mail_temp_dir``.
+This decompression uses a temporary that is written to ``mail_temp_dir``. By
+using the ``compress`` fs wrapper in ``obox_fs`` line the mails are stored
+uncompressed in fscache, and reading the mails from there doesn't require
```

Comment:
This is only true if compress is specified after fscache, not true if before. Should also clarify that here now that it's being changed anyway..

Reply:
Right.  s/wrapper/wrapper after ``fscache``/